Makefile

```diff
@@ -186,14 +189,43 @@ docclean:
 
 RAWVER=$(shell $(PYTHON) tools/getnodeversion.py)
 VERSION=v$(RAWVER)
+ifdef NIGHTLY
```

Comment:
Based on your explanation, I think something like BUILDTYPE might be more appropriate for this parameter.